locales/fr.lua

```diff
@@ -860,7 +860,7 @@ for key,value in pairs({
 		--TODO: [-1202] = "Fyrakk Assaults",						-- Fyrakk Assaults
 		--TODO: [-1203] = "Sniffenseeking",							-- Sniffenseeking
 	-- Warrior order hall lore items
-		--TODO: [-2200] = "Great Odyn and the Firelord",
+		[-2200] = "Le grand Odyn et le seigneur du feu",
```

Comment:
Did you get this one in-game? Just wondering if you also have the rest from below, they are on the same object.